tests/examples/test_examples_run.py

```diff
@@ -9,7 +9,8 @@
 examples_dir = join(cwd, "..", "..", "examples")
 example_files = glob.glob("%s/*.py" % examples_dir)
 
-@pytest.fixture(params=[pytest.param(f, marks=pytest.mark.xfail(reason="unreliable")) if f.endswith("h_mountain.py") else f for f in example_files],
+
+@pytest.fixture(params=glob.glob("%s/*.py" % examples_dir),
```

Comment:
Is the xfail marking being removed because `mountain_hydrostatic.py` is now expected to pass?

Reply:
Yes, because I think it should pass. I need to verify this though.

Reply:
Right now, I'm fighting with Jenkins...

Reply:
OK, I think I still expect it to fail unless it's been updated?

Reply:
Well I expect it to fail sometimes on Jenkins but to pass for everyone else...

Reply:
You're right. It still fails. This is a bad test case. Still don't know why and it's so extremely irritating.

Reply:
I suggest you just put this xfail line back in and then at least it will pass!

Reply:
I'm pretty confident this will pass now.